src/main/java/onehundredandfour/GameHelper.java

```diff
@@ -28,164 +53,198 @@
+		List<Player> playersInGame = new ArrayList<Player>(players);
+		Player playerWithLowestCard;
+		for (Player player : playersInGame) {
+			if (player.getCardOnTable() == null) {
+				playersInGame.remove(player);
 			}
 		}
-		Player min = p.get(0);
-        for (Player i : p){
-            min = min.getCardOnTable().getValue() < i.getCardOnTable().getValue() ? min : i;
-        }
-		return min;
+		if (playersInGame.size() < 1) {
+		playerWithLowestCard = playersInGame
+			      .stream()
+			      .min(Comparator.comparing( plyr -> plyr.getCardOnTable().getValue()))
+			      .orElseThrow(NoSuchElementException::new);
+		}else {playerWithLowestCard = playersInGame.get(0);}
+		return playerWithLowestCard;
 		}
 	
-//	public static boolean checkCardAgainstRows(Card card, Game game) {
-//		List<Card> after = new ArrayList<>();
-//
-//		if (card.getValue() > game.getRow0().get(game.getRow0().size()-1).getValue()) {
-//		after.add(0, game.getRow0().get(game.getRow0().size()-1));	
-//		game.getRow0().add(card);
-//		}
-//		if (card.getValue() > game.getRow1().get(game.getRow1().size()-1).getValue()) {
-//			if (after.isEmpty()) {
-//				after.add(game.getRow1().get(game.getRow1().size()-1));	
-//			}else if (game.getRow1().get(game.getRow1().size()-1).getValue() > after.get(0).getValue()) {
-//				game.getRow0().remove(card);
-//				game.getRow1().add(card);
-//			}
-//		}
-//		if (card.getValue() > game.getRow2().get(game.getRow2().size()-1).getValue()) {
-//			if (after.isEmpty()) {
-//				after.add(game.getRow2().get(game.getRow2().size()-1));
-//			}else if(game.getRow2().get(game.getRow2().size()-1).getValue() > after.get(0).getValue()) {
-//				game.getRow0().remove(card);
-//				game.getRow1().remove(card);
-//				game.getRow2().add(card);
-//			}
-//		}
-//		if (card.getValue() > game.getRow3().get(game.getRow3().size()-1).getValue()) {
-//			if (after.isEmpty()){
-//				after.add(game.getRow3().get(game.getRow3().size()-1));	
-//			}else if (game.getRow3().get(game.getRow3().size()-1).getValue() > after.get(0).getValue()) {
-//				game.getRow0().remove(card);
-//				game.getRow1().remove(card);
-//				game.getRow2().remove(card);
-//				game.getRow3().add(card);
-//			}
-//		}
-//		if (!after.isEmpty()) {
-//			return true;
-//		}else
-//		{	System.out.println("Card is lower than last cards in all rows. \r\nPlease pick a row.");
-//			return false;
-//		}
-//	}
-	
-	public static boolean checkPlayerCardAgainstRows(Player pCard, Game game) {
+	/**
+	 * checkPlayerCardAgainstRows will validate the card of the player against the existing rows in the game.
+	 * 
+	 * @param player the player for which the card needs to be validated.
+	 * @param game the game for which the card is validated for.
+	 * @return boolean on whether the card was successfully added to the game.
+	 */
+	public static boolean checkPlayerCardAgainstRows(Player player, Game game) {
```

Comment:
Just a thought. Would it help to create a list of the four rows and iterate through them to reduce the number of lines in the code?
example:
for (List<Card> row : Arrays.asList(game.getRow0(), game.getRow1(), game.getRow2(), game.getRow3())) {
  doStuffPerRow;
}